website/content/docs/commands/namespace.mdx

```diff
@@ -44,10 +68,12 @@ Usage: vault namespace <subcommand> [options] [args]
   current logged in token belongs to.
 
 Subcommands:
-    create    Create a new namespace
-    delete    Delete an existing namespace
-    list      List child namespaces
-    lookup    Look up an existing namespace
+    create            Create a new namespace
+    delete            Delete an existing namespace
+    list              List child namespaces
+    lookup            Look up an existing namespace
+    api-lock/lock     Lock the API for a namespace
+    api-lock/unlock   Unlock the API for a namespace
```

Comment:
The `api-lock/` prefix here is extraneous, correct? I was going to provide a suggestion but it likely would have affected the formatting.

Reply:
oh in the command section it is, good catch! afb0dd3c1